en/guide/routing.md

```diff
@@ -1,14 +1,81 @@
 # Routing
 
-MAVLink 2.0 has a source system and component ID as well as a destination system and component ID. While the source IDs are part of every message, destination IDs are only assigned for cases where a destination ID is required.
+This topic explains how messages should be routed by MAVLink systems. 
 
+## Overview
+
+A MAVLINK network is made up of systems (vehicles, ground stations, antenna trackers, cameras etc.) that are composed from components (autopilot, camera, servos, etc.).
+
+Each system has a network-unique *system id*, and each component has a system-unique *component id* that can be used for addressing/routing:
+- The *system id* has a value between 1 and 255. 
```

Comment:
As discussed offline, it is required because broadcast messages are defined as either not having a `system_id` or having a `system_id` of 0. As things are now setting an id of 0 would be a problem if you sent a message that was expected to only be acted on by that system. Also the message would be routed even when you don't want it to be.

So GCS should disallow 0 being set.

IMO it would make sense for *just* omission of a system id to indicate broadcast so that id of 0 could be used for a system too. But I think the effort to untangle this would be more effort than the gain of an extra id and consistency.